primer.md

```diff
@@ -174,6 +175,13 @@ attributes may be defined that facilitate meeting persistence requirements but
 it is expected that these will continuously evolve along with industry best
 practices and advancements.
 
+The Cloudevents spec will not mandate any vendor specific Authorisation model, As the
```

Comment:
- Can you wrap these at 80 columns?
- s/, As/, as/

Reply:
done